src/app/saga/manuscript.saga.ts

```diff
@@ -18,6 +17,6 @@ export function* loadManuscriptSaga(action: Action<string>) {
 export default function* () {
   yield all([
     takeLatest(manuscriptActions.loadManuscriptAction.request.type, loadManuscriptSaga),
-    takeLatest(initActions.initApplication.type, loadManuscriptSaga)
+    takeLatest(initActions.loadArticle.type, loadManuscriptSaga)
```

Comment:
those actions looks the same to me. Why do we call load manuscript saga on 2 different actions?